pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -431,13 +445,16 @@ public void callingWithBannedNodeidsOptionButNoValueMustDisplayErrorAndUsage() {
 
   @Test
   public void bootnodesOptionMustBeUsed() {
-    final String[] nodes = {"enode://001@123:4567", "enode://002@123:4567", "enode://003@123:4567"};
+    final String[] nodes = {
+      "enode://001@192.168.0.1:4567", "enode://002@192.168.0.1:4567", "enode://003@192.168.0.1:4567"
+    };
```

Comment:
aren't these also invalid because the 001 part is an odd number of characters - should be a hex string?